cmake_files/dependencies.cmake

```diff
@@ -22,7 +22,7 @@ endif()
 if(openmp)
   find_package(OpenMP)
   if(OPENMP_FOUND)
-    set(SOPT_DEFAULT_OPENMP_THREADS 4 CACHE STRING "Number of threads used in testing")
+    set(SOPT_DEFAULT_OPENMP_THREADS 2 CACHE STRING "Number of threads used in testing")
```

Comment:
I'm just checking what this does on my laptop, I'll let you know in the meeting tomorrow.

Reply:
Didn't make a difference on mine either.